File: 7KYU/Beginner Series #3 Sum of Numbers/solution.cs
using System;
using System.Collections.Generic;
using System.Linq;
public class Sum
{
	public int GetSum(int a, int b)
	{
		Console.WriteLine($"{a} {b}");
		var total = 0;
		var range = GetRange(a, b);
		for(var i = range.Start; i <= range.End; i++)
			total+= i;
		return total;
	}
     
	public (int Start, int End) GetRange(int a, int b)
		=> (a > b)
			   ? (b, a)
			   : (a, b);
}

File: 7KYU/Beginner Series #3 Sum of Numbers/solution.cpp
int get_sum(int a, int b)
{
  int start = a, end = b;
  
  if(a > b) {
    start = b;
    end = a;
  }
  
  int total = 0;
  for(int i = start; i <= end; i++){
    total += i;
  }
  return total;
}